content/raw/wallet/2.0.0_send_transaction.md

```diff
@@ -0,0 +1,142 @@
+---
+title: "Send Transaction spec"
+description: ""
+version: "2.0.0"
+date: 2022-05-10T12:03:57+00:00
+lastmod: 2022-05-10T12:03:57+00:00
+draft: false
+images: []
+menu:
+  raw:
+    parent: "wallet"
+weight: 100
+toc: true
+---
+
+# Send Transaction Spec
+
+## Motivation
+
+One of the most common use cases for a wallet is to send tokens to another account. A user typically expects to know how much tokens they are sending are worth in some currency and also expect a wide range of functionality such as ENS addresses, address validation using EIP-55, and the ability to choose easily common addresses, saved or recently used address to send to. The user also expects to be able to understand the transactions fees involved and choose the priority for transaction.
+
+## Definitions
+
+### Best route
+
+We define the best route as the less costly route amongst all the possible route
+
+### Send
+
+Send is the action of creating one or more transactions in order to send token from an account to another account
+
+### Swap
+
+Swap is the action of creating one or more transactions in order to swap token from another token to the same account
+
+### Bridge
+
+Swap is the action of creating one or more transactions in order to bridge a token from one chain to another chain
+
+### Adddress field
+
+The address field can be filled with multiple option:
+1. an ethereum address
+2. an ENS name
+3. a Contact display name
+
+Plus it can be prefixed by the destination chains of the transaction separated by `:`
```

Comment:
will we write down the various destinations chains supported in the feature specs?

Reply:
Yes I need to add specs for chain supported and bridge supported